src/Icicle/Avalanche/Statement/Simp/Dead.hs

```diff
@@ -59,11 +59,15 @@ deadS us statements
         in  if   usedX n sU
             then (mconcat [xU, sU], sK, Let n x sS)
             else (sU, sK, sS)
-    ForeachInts n from to ss
+    While t n end ss
+     -> let (sU, sK, sS) = deadLoop us ss
+            eU           = usageX end
+        in (addUsedA n $ mconcat [sU, eU], sK, While t n end sS)
```

Comment:
I think this is equivalent to `mconcat [sU, eU, usageA n]`. is this faster?
(nitpick)